src/fileurlreader.cpp

```diff
@@ -23,16 +23,15 @@ nonstd::optional<std::string> FileUrlReader::reload()
 	tags.clear();
 	alltags.clear();
 
-	std::fstream f;
```

Comment:
Can we drop `#include <fstream>` now?

Reply:
No, the `FileUrlReader::write_config()` method still uses an `std::fstream`:
https://github.com/newsboat/newsboat/blob/f2c16f01ebd839e6e7e32dcbd7d8759449bf20b2/src/fileurlreader.cpp#L63-L64

Reply:
Ah, okay then. Thanks!